src/bridge/ui_commands.rs

```diff
@@ -3,7 +3,9 @@ use nvim_rs::compat::tokio::Compat;
 use nvim_rs::Neovim;
 use tokio::process::ChildStdin;
 
+#[cfg(not(feature = "glfw"))]
```

Comment:
This could possibly wait for #347 to be merged in.

Reply:
https://github.com/Kethku/neovide/pull/364/commits/a97fb234ea7eeefc5c8f0f3b89f54b8f227d6da4